File: test/ocamlTests/CharTest/isPrintableTest.ml
test "isPrintable('~')" (fun () -> expect (Char.isPrintable '~') |> toEqual Eq.bool true) ; 
test "isPrintable(fromCode(31) |> Option.map(~f=isPrintable))" (fun () -> expect (Char.isPrintable fromCode(31) |> Option.map(~f=isPrintable)) |> toEqual (let open Eq in bool ) Some(false)) ; 

File: test/fsharpTests/CharTest/isPrintableTest.fs
testCase "isPrintable('~')" 
<| fun _ -> 
    let expected = true
    Expect.equal expected (Char.isPrintable '~') "error"
testCase "isPrintable(fromCode(31) |> Option.map(~f=isPrintable))" 
<| fun _ -> 
    let expected = Some(false)
    Expect.equal expected (Char.isPrintable fromCode(31) |> Option.map(~f=isPrintable)) "error"
